.github/workflows/on-push.yml

```diff
@@ -52,8 +52,8 @@ jobs:
     - name: Install python dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r Browser/requirements.txt
         pip install -r Browser/dev-requirements.txt
+        pip install -r Browser/requirements.txt
```

Comment:
Dev requirements automation run normal requirements, see in https://github.com/MarketSquare/robotframework-browser/blob/master/Browser/dev-requirements.txt#L22